.github/project.yml

```diff
@@ -1,4 +1,4 @@
 name: SmallRye Config
 release:
-  current-version: 2.8.1
-  next-version: 2.8.2-SNAPSHOT
+  current-version: 2.9.0
```

Comment:
Are we sure we want to bump the minor version here?

Reply:
2.9.0 sounds correct to me.

Reply:
No, I was targeting 2.8.2, but that was before the otel/mptel upgrade, so 2.9.0 is probably fine.